### PR TITLE
Rename GitHub Actions Jobs

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: .github/workflows/groovy_checkstyle.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on: [push]
 
 jobs:
   build:
+    name: checkstyle
```
Comment (jenniferboedker) [inline .github/workflows/groovy_checkstyle.yml:7]: wait, you need to update the 'build' one line above, right?

Comment (KochTobi) [reply .github/workflows/groovy_checkstyle.yml:7]: No this works too 😄

Comment (KochTobi) [reply .github/workflows/groovy_checkstyle.yml:7]: I think I did not need to change this actions name but I did it anyways :)

Comment (jenniferboedker) [reply .github/workflows/groovy_checkstyle.yml:7]: but it does not make sense to me to call it "build" in the line above but give it another name afterwards ... more code where not necessary.

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**